feature-libs/quote/core/connectors/quote.adapter.ts

```diff
@@ -5,52 +5,55 @@
  */
 
 import { PaginationModel } from '@spartacus/core';
-import { Observable } from 'rxjs';
 import {
-  QuoteList,
-  QuoteStarter,
+  Comment,
   Quote,
-  QuoteMetadata,
   QuoteActionType,
-  Comment,
   QuoteDiscount,
+  QuoteList,
+  QuoteMetadata,
+  QuoteStarter,
 } from '@spartacus/quote/root';
+import { Observable } from 'rxjs';
 
 export abstract class QuoteAdapter {
   /**
-   * Abstract method used to list of quotes for user
+   * Fetches a list of quotes
    *
-   * @param userId
-   * @param pagination
+   * @param userId Quote user
+   * @param pagination Pagination options
+   * @returns Observable emitting a quote list
    */
   abstract getQuotes(
     userId: string,
     pagination: PaginationModel
   ): Observable<QuoteList>;
 
   /**
-   * Abstract method used to request / create a quote
-   *
-   * @param userId
+   * Creates a quote
+   * @param userId Quote user
+   * @param quoteStarter Attributes needed to create a quote
+   * @returns Observable emitting a quote
    */
   abstract createQuote(
     userId: string,
     quoteStarter: QuoteStarter
   ): Observable<Quote>;
 
   /**
-   * Abstract method used to get quote details
-   *
-   * @param userId
-   * @param quoteCode
+   * Gets a quote specified by its ID (aka code)
```

Comment:
Could we use `retrieves` for **getter** instead of `fetches / gets` ...?

Reply:
Changed to 'fetches', see https://thecontentauthority.com/blog/fetch-vs-retrieve